### ISSUE TITLE
Getting ANSI start and end sequence separated

### ISSUE THREAD

**Issue by dominikwilkowski** [jam1garner/owo-colors#57]
Is there a way to get the start and end ANSI sequence from owo_colors for a given rgb value (or any other parsed color I guess)?
I need it and before implementing a method to generate a color, split if and take each bit separately I thought I ask here.
In my mind it would be something like this:
```rust
println!("unstyled {} some styled text here {} unstyled",
  OwoColorize.rgb(70, 130, 180).start,
  OwoColorize.rgb(70, 130, 180).end
);
```
Or something like that?

**Comment by jam1garner** [comment]
This should be resolved by #55, although it hasn't been included in a release just yet. Within the next week I expect it to be on crates.io, planning on making a new minor release after #56

**Comment by jam1garner** [comment]
Closing, @dominikwilkowski, as it should be included in #55. See https://docs.rs/owo-colors/latest/owo_colors/struct.Style.html#method.fmt_prefix
